src/main/java/spring/service/impl/UserServiceImpl.java

```diff
@@ -23,4 +24,9 @@ public void add(User user) {
     public List<User> listUsers() {
         return userDao.findAllUsers();
     }
+
+    @Override
+    public Optional<User> getUserById(Long userId) {
```

Comment:
let's return user without optional at the service layer

Reply:
done